www/source/partials/docs/_dev-pkgs-composite-pkg.html.md.erb

```diff
@@ -0,0 +1,121 @@
+# <a name="composite-pkgs" id="composite-pkgs" data-magellan-target="composite-pkgs">Creating Composite Packages</a>
+
+Composites are another type of Habitat package, but one that is all metadata.  They allow you to group together services that should be run together on the same Supervisor, enabling you to take better advantage of modern deployment patterns with your Habitat services. 
+
+> **Note** The composite functionality is still being updated and refined. It is not recommended to run this as part of production workflows at this time.
+
+Because these services use the same Supervisor, this also means they are running on the same machine. They are related services, but they are distinct enough to remain separate; they run separate processes, have separate development lifecycles, and so on. Examples of this pattern include: database and archiver (such as [WAL-E](https://github.com/wal-e/wal-e) for Postgres), service and a service mesh proxy (such as [Envoy](https://github.com/envoyproxy/envoy)), and service and log aggregation agent.
+
+Like other packages, composite packages are built from a plan.sh file. 
+
+~~~
+pkg_origin="yourorigin"
+pkg_name="composite-example"
+pkg_type="composite"
+pkg_version="0.1.0"
+
+pkg_services=(
+    origin/package1
+    origin/package2
+)
+
+pkg_bind_map=(
+    [origin/package2]="http:origin/package1"
+)
+~~~
+
+The `pkg_origin`, `pkg_name`, and `pkg_version` settings serve the same role they do in other Habitat packages, but `pkg_type`, `pkg_services`, and `pkg_bind_map` are only specific to composite packages. 
+
+Plans for composite packages must have the `pkg_type` setting set to the string `"composite"`. Without `pkg_type` defined, the package is assumed to be a "standalone" package type, which is the default type for non-composite packages. 
+
+The `pkg_services` array is where you enumerate all the services that are in your composite package. Services are defined as having a binary that gets executed via a `run` hook. By default, these should be either packages you have built and/or installed on your local workstation, or packages that are in the `stable` channel in Builder. If you wish to use packages that are in the `unstable` channel, specify `HAB_BLDR_CHANNEL=unstable` in your environment when you build your composite package.
+
+> **Note** You must specify at least two services in the `pkg_services` array. 
+
+The last composite-specific setting, `pkg_bind_map`, lets you declare all the binding relationships between all the services in your composite package. In the example above, `origin/package2` has a bind named `http` that is satisfied by the `origin/package1` service. Additional binds can be added by separating them with spaces, for example `"http:origin/package1 otherbind:origin/package3"`.
+
+## Using Composite Packages
+To use a composite package:
+
+* Build it in Builder or in a local Studio environment as you would any other package
```

Comment:
I'm not actually 100% sure that we can build composites in Builder at the moment: #3801